python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -146,13 +154,132 @@ def grow_exp(df_to_grow: pd.DataFrame, num_times_xx2: int):
         df_to_grow = pd.concat([df_to_grow, df_prev])
     return df_to_grow
 
+def generate_big_dataframe(rows:int=1000000):
+    print("Generating big dataframe")
+    st = time.time()
+    df = get_sample_dataframe(rows)
+    df = grow_exp(df, 5)
+    print("Generation took :", time.time() - st)
+    return df
+
+#endregion
+
+#region HELPER functions for memray tests
+
+def construct_df_querybuilder_tests(size: int) -> pd.DataFrame:
+    df = get_sample_dataframe(size)
+    date_range = pd.date_range(start="2000-1-1", periods=size, freq="s")
+    df.index = pd.Index(date_range)
+    df.index.name = "timestamp"
+    return df
+
+
+def query_group() -> QueryBuilder:
```

Comment:
This name is a bit misleading, as the generated query will do a filter, then a groupby

Reply:
The name still doesn't mention filtering